src/components/order-service/order-service.controller.ts

```diff
@@ -12,10 +12,12 @@ export class OrderServiceController implements angular.IController {
   }
 
   public $onInit() {
-    this.ctrl.serviceIcon = this.ctrl.service.imageURL ? this.ctrl.service.imageURL : 'pf pficon-service';
-    this.ctrl.serviceName = this.ctrl.service.displayName ? this.ctrl.service.displayName : this.ctrl.service.metadata.name;
-    this.ctrl.versions = this.ctrl.service.versions;
-    this.ctrl.selectedVersion = this.ctrl.service.versions[0];
+    this.ctrl.serviceIcon = this.ctrl.service.icon ? this.ctrl.service.icon : 'pf pficon-service';
```

Comment:
I see where this was fixed in the models and for the tiles, but in the order panel it looks like its still just using the class-based approach and not handling both ways.